source/lib/repositories-service.js

```diff
@@ -0,0 +1,47 @@
+import {parseLinkHeader, parseFullName} from '../util';
+import repositoriesStorage from '../repositories-storage';
+import {makeApiRequest} from './api';
+
+export async function getRepositories(
+	repos = [],
+	params = {
+		page: '1',
+		per_page: '100' // eslint-disable-line camelcase
+	}
+) {
+	const {headers, json} = await makeApiRequest('/user/repos', params);
```

Comment:
This should be wrapped inside a `try {} catch {}` block, as we cannot know when and why a request might fail.

Also bubbling up the error to the user would be a massive UX as well. 👍

Reply:
Very true - but what exactly do you mean by `would be a massive UX as well`?

Reply:
If for any reason, the requests fail or cache fails to load, we could inform the user to something like "try again".